code/src/index.css

```diff
@@ -1,13 +1,135 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: 'Roboto', sans-serif;
+  background-color: black;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+/* Movie list */
+
+.movie-container {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  flex-flow: wrap;
+}
+
+.movie {
+  flex-basis: 25%;
+}
+
+.movie img {
+  width: 100%;
+}
+
+.movie-overlay {
+ position: relative;
+}
+
+.text-overlay { 
+  position: absolute;
+  bottom: 0;
+  color: #fff;
+  background: rgba(0, 0, 0, 0.742);
+  width: 100%;
+  height: 100%;
+  transition: 0.3s ease;
+  opacity: 0;
+}
+
+.text-overlay h1 { 
+  position: absolute;
+  bottom: 50px;
+  margin-left: 20px;
+  font-size: 30px;
+}
+
+.text-overlay p {
+  position: absolute;
+  bottom: 0;
+  padding: 20px;
+  font-size: 17px;
+}
+
+.text-overlay:hover {
+  opacity: 1;
+}
+
+/* Individual movie details*/
+
+.backdrop-container {
+  width: 100%;
+  min-height: 100vh;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.movie-details-container {
+  min-height: 90vh;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
 }
+
+.movie-details {
+  color: white;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  width: 100%;
+  padding: 20px;
+}
+
+.poster {
+  width: 400px;
+  height: 500px;
+  margin: 5px;
+  border-style: solid;
+  border-color: white;
+}
+
+.details {
+  width: 50%;
+}
+
+.details h1 {
+  font-size: 48px;
+}
+
+.rating {
+  color: green;
+  font-size: 24px;
+}
+
+.button-container {
+ 
+  height: 80px;
+  width: 80px;
+  margin-left: 10px;
+  
+}
+
+.back-button {
+  transform: translate(100%, 91%);
+  width: 50px;
+  height: 50px;
+}
+
+/* mobile view */
+
+@media only screen and (max-width: 400px) {
```

Comment:
It is good that you think to make mobile responsive even it is not requirement 👍